gspread/models.py

```diff
@@ -48,7 +48,8 @@
 class ValueRange(list):
     @classmethod
     def from_json(cls, json):
-        new_obj = cls(json['values'])
+        values = json['values'] if 'values' in json else []
```

Comment:
I think we can improve it even further if we use `dict.get` with default, e.g. `json.get('values', [])`

Reply:
Great idea, done in 903f268